openpgp.d.ts

```diff
@@ -643,8 +630,9 @@ export interface FullKeyOptions {
   subkeys: KeyOptions[]; // required unlike KeyOptions.subkeys
 }
 
-export interface Keyid {
+export class KeyId {
   bytes: string;
+  equals(keyid: KeyId, matchWildcard?: boolean): boolean;
```

Comment:
By far the most commonly used method here should be `toHex`, maybe you want to add that one?